src/main/java/org/openhab/tools/analysis/report/ReportUtility.java

```diff
@@ -286,8 +286,7 @@ private String checkForErrors(String secondMergeResult, String reportLocation) {
                 result.append(log);
             }
         }
-        result.append("Detailed report can be found at: file").append(File.separator).append(File.separator)
-                .append(File.separator).append(reportLocation).append("\n");
+        result.append("Detailed report can be found at: file:///").append(reportLocation).append("\n");
```

Comment:
Relay three slashes aren't there two slashes between the scheme and the location and normally the first one if the location is an absolute one starting from the root (but part of the location)?